gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -436,6 +436,10 @@ public class GoogleHadoopFileSystemConfiguration {
   public static final HadoopConfigurationProperty<Boolean> GCS_TRACE_LOG_ENABLE =
       new HadoopConfigurationProperty<>("fs.gs.tracelog.enable", false);
 
+  /** Configuration key to enable using google-cloud-storage client for connecting with GCS */
+  public static final HadoopConfigurationProperty<Boolean> GCS_MANUAL_CLIENT_ENABLE =
+      new HadoopConfigurationProperty<>("fs.gs.manualclient.enable", false);
```

Comment:
rename this to fs.gs.gcs.javaclient.enable? "manual" do not sound right.

Reply:
sure.